src/buttons/test/buttons.spec.js

```diff
@@ -201,7 +201,7 @@ describe('buttons', function () {
         expect(btns.eq(1)).not.toHaveClass('active');
 
         btns.eq(0).click();
-        expect($scope.model).toEqual(undefined);
+        expect($scope.model).toBeNull();
```

Comment:
It says "undefined is not equal to null" so it is null. I guess Jasmine is more picky with undefined/null.